src/plugins/data/public/search/expressions/esaggs.ts

```diff
@@ -149,7 +149,12 @@ const handleCourierRequest = async ({
   request.stats(getRequestInspectorStats(requestSearchSource));
 
   try {
-    const response = await requestSearchSource.fetch({ abortSignal });
+    const response = await requestSearchSource.fetch({
+      abortSignal,
+      // TODO: In order to run expressions outside a session, the session ID should be passed as input
+      // Issue:
```

Comment:
I think we need an issue linked here :D

Reply:
I actually prefer to remove the link altogether. When it's time to do it, we'll find it.